### DISCUSSION TITLE
Groups variable in Flax Conv layers

### DISCUSSION THREAD

**Discussion by conceptofmind** [google/flax#2175] [Q&A] [answered]
Hi,

I am attempting to refactor a piece of code from Tensorflow to Flax. In Tensorflow we are able to explicitly define *groups*, `groups=hidden_dim`, in Conv2D() layers. Is there a way to define the same *groups* attribute/argument in Flax Conv() layers?

For example converting a convolutional layer from Tensorflow to Flax:

Tensorflow
```python
x = nn.Conv2D(filters=hidden_dim, kernel_size=3, strides=stride, padding='SAME', groups=hidden_dim, use_bias=False)(x)
```

Flax
```python
x = nn.Conv(features = hidden_dim, kernel_size = 3, strides = stride, padding = 'SAME', *groups=*, use_bias = False)(x)
```

Flax has a *feature_group_count*. Is that the same thing as Tensorflow *groups*? I tried to review the documentation but the section for the Conv docs page doesn't show attribute/argument descriptions.

Any help would be greatly appreciated.

Thank you,

Enrico

**Comment by jheek** [comment, marked as answer]
Yes feature_group_count should have the same effect as groups.

**Reply by conceptofmind** [reply]
Hi @jheek ,

Thank you for clarifying.

Enrico
